package-lock.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "lorawan-devices",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
@netvox-tech please run `make deps` so this file doesn't get any modification

Reply:
OK, I will do that

Reply:
You're probably using an older Node version. We're using Node 16 and npm 8.

Reply:
I have updated Node, npm, and go. It seems that the problem is fixed.
Thanks.